### DISCUSSION TITLE
Wazuh agent duplicated IP

### DISCUSSION THREAD

**Discussion by tanitsu** [Security-Onion-Solutions/securityonion#9580] [Unsupported Versions]
Hello,

I hop you can help me.
I have a master accessible from internet  and 2 assets to manage.
The 2 assets are on the same local network with the same internet connection so the same public IP.

When I try to add the agents I need to specify the agent name and the IP (so public IP that the master will receive ?) I have a duplicate IP error ( logic )

but to you have a solution for my usecase ?

thanks for your help.
Regards

**Comment by cm-ops** [comment]
Have you tried Wazuh agent's IP address as `any `? Also, changing  `use_source_ip` to `no` as described here https://documentation.wazuh.com/3.13/user-manual/registering/registering-agents-troubleshooting.html
